models/Participant.model.js

```diff
@@ -1,13 +1,17 @@
 const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
-const participantSchema = Schema(
-  {
+const participantSchema = Schema({
+  userId: Number,
```

Comment:
Another place I think using `owner` is better. My bad for not mentioning this earlier.
```js
owner: {
    ref: "User",
    required: true,
    type: Schema.Types.ObjectId,
  },
```
Im not sure if you've already encountered this in class.